tests/test_routes.py

```diff
@@ -18,14 +18,6 @@ def test_homepage(self):
 
         self.assertEqual(self.client.get("/").status_code, 200)
 
-    def test_reference(self):
-        """
-        When given the ubuntu-frame URL,
-        we should return a 200 status code
-        """
-
-        self.assertEqual(self.client.get("/reference").status_code, 200)
-
```

Comment:
I think you can also keep this test, but change it for `/docs` it could be useful. And if you want, you can also add more for the other pages